views/partials/beerpartial.hbs

```diff
@@ -0,0 +1,17 @@
+{{#each beersArray}}
```

Comment:
this part `{{#each beersArray}}` should be a part of the beer.hbs file. Since it's supposed to display all the beers there, that's where you should iterate over them. The partial is only one of them.